streaming/src/main/scala/com/stratio/crossdata/streaming/helpers/CrossdataStreamingHelper.scala

```diff
@@ -38,6 +38,9 @@ object CrossdataStreamingHelper extends SparkLoggerComponent {
                     crossdataCatalogConf: Map[String, String]): StreamingContext = {
     val sparkStreamingWindow = ephemeralTable.options.atomicWindow
     val sparkContext = SparkContext.getOrCreate(sparkConf)
+
+    val countdowns = collection.mutable.Map[String, Int]()
```

Comment:
An accumulator is created from an initial value v by calling SparkContext.accumulator(v). Tasks running on the cluster can then add to it using the add method or the += operator (in Scala and Python). However, they cannot read its value. Only the driver program can read the accumulator’s value, using its value method.